jetson/filter/src/__main__.py

```diff
@@ -356,6 +363,24 @@ async def run(self):
                                                         bearing,
                                                         ref_lat=self.config["RefCoords"]["lat"],
                                                         ref_long=self.config["RefCoords"]["long"])
+
+                # Simulates a fixed position at the TargetCoords specified in config,
+                # only passes through valid bearing data
+                elif self.config["FilterType"] == "BearingOnlyPipe":
+                    bearing = self._getFreshBearing()
+                    if bearing is None:
+                        continue
+                    decimal_pos = self.config["TargetCoords"]
```

Comment:
Are we able to change retrieving the TargetCoords properly v. retrieving the TargetCoords from the config file, by changing a build flag or the config file?